lib/index.js

```diff
@@ -19,6 +19,11 @@
  *   You should typically pass `React.Fragment`.
  * @property {string|undefined} [prefix='h-']
  *   React key prefix.
+ * @property {boolean|undefined} [mapDepricatedTableCellAttrsToInlineCss=true]
+ *   Default: true. Map deprecated table-related attributes to css styles as
+ *   recommended by Mozilla Developer Network docs. E.g.
+ *   `align="center"` to `style="text-align: center;"`. May result
+ *   in rendering differences.
```

Comment:
Some recommendations:

```suggestion
 *   Fix obsolete align attributes on table cells by turning them into inline
 *   styles.
 *   Keep it on when working with markdown, turn it off when working with
 *   markup for emails.
```

Feel free to iterate on this.

* Uses [semantic line breaks](https://sembr.org)
* `default: true` is already added with the `[xxx=true]` notation
* the goal of this feature is to solve obsolete attributes, not just because MDN says it too
* adds a recommendation on when to have it on/off

Reply:
I accepted this change with the exception that I kept the `Default: true` comment at the end. When typescript processes this comment and creates the `SharedOptions` interface, it keeps the JSDoc comments but doesn't keep any reference to `[fixTableCellAlign=true]` (so, when inspecting the type in the editor, you don't know that it has a default value). By also referencing the default value in the comment, we ensure that the generated type retains this information.

For example, here's what I see when I inspect the `prefix` option in VS Code. There is no reference to a default value in the editor tooltip and there also isn't a reference to it in the generated typescript type.

<img width="984" alt="Screenshot 2023-04-13 at 11 08 38 AM" src="https://user-images.githubusercontent.com/12599934/231836729-69b50bdc-b242-40d4-b89d-85ebe93440e3.png">